src/App.tsx

```diff
@@ -1,14 +1,47 @@
 /* eslint-disable max-len */
-import React from 'react';
+import {
+  useState,
+  useEffect,
+  useMemo,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { Todo } from './types/Todo';
+import { FilterOptions } from './types/FilterOptions';
+import { getTodos } from './api';
+import { Loader } from './components/Loader';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
-import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [currentTodo, setCurrentTodo] = useState<Todo | null>(null);
+  const [query, setQuery] = useState<string>('');
+  const [filterOption, setFilterOption] = useState<string>(FilterOptions.all);
```

Comment:
this is not just any string
```suggestion
  const [filterOption, setFilterOption] = useState<FilterOptions>(FilterOptions.all);
```